### PR TITLE
Adding popup window

### PR DESCRIPTION
**Pull request by edahigure**
In this branch, we have implemented a pop-up window showing information about the projects when the user clicks the buttons.

### CODE REVIEW

File: project.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+            <div class="button1">${card.technologies[3]}</div>                                
+        </div>  
+
+        <div class="wrapper-3">        
+            <div class="red"> 
+                <img src="${card.image}" class="image-work-popup">
+            </div>
+            <div class="wrapper-4">                
+                <div class="brown"> 
+                   <p class="works-description-popup">
```
Comment (jerryowusu) [inline project.js]: Kindly ensure that the popup follows the design for both the desktop and mobile versions from the chosen template in figma. :+1: 

- [ ] Kindly fix the width and height of the pop-up for both versions of mobile and desktop. It should also be in the center and responsive.
- [ ] Check the position of the `languages`, the project image and the `see source button` etc.



Your mobile pop-up            |  The figma template
:-------------------------:|:-------------------------:
Screenshot from 2022-10-21 22-15-00  |  Screenshot from 2022-10-21 22-16-46


Your desktop pop-up             |  The figma template
:-------------------------:|:-------------------------:
Screenshot from 2022-10-21 22-15-42  |  Screenshot from 2022-10-21 22-14-37

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @edahigure 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few issues to need to work on and make your project look professional :sunglasses:

soclose


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good commit messages
:heavy_check_mark: Follow correct GitHub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Kindly follow the readme [template](https://github.com/microverseinc/readme-template/blob/master/README.md), ensure that all sections and headers are available to make your project look more proffessional and up to standard

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -1823,60 +1906,321 @@ a {
...
+    top: 10px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    z-index: 2;
+  }
+
+  .wrapper-0 {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (Santiago220991) [inline style.css]: As [the last code reviewer suggested](https://github.com/edahigure/Portfolio-setup-and-mobile-first/pull/13#discussion_r1002221758) kindly fix the position and the size of your popup windows on both versions mobile and desktop. :pray: :pray: 
- Please apply the changes in your desktop version and check it with your browser window zoom at 100%. :eyes: :eyes: 
Your design
image
Figma design
image

- The design in your mobile version is totally different, kindly place the elements in the same way that the Figma template. :+1: :smiley: :smiley: 
Your design.
image

Figma design
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @edahigure, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
:heavy_check_mark: No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @edahigure, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly give proper margin & design to the marked place in the picture.
Screenshot from 2022-10-22 23-33-47

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**